src/screens/IndexScreen.js

```diff
@@ -19,13 +19,15 @@ const IndexScreen = ()  => {
                 //function arg that has object that is equal to that object
                 renderItem={({item}) => {
                     return (
-                        <View style={styles.row}>
-                            <Text style={styles.title}>{item.title} - {item.id}</Text>
-                            <TouchableOpacity onPress={() => deleteBlogPost(item.id)}>
-                                <Feather style={styles.icon} name="trash"/>
-                            </TouchableOpacity>
-                            
-                        </View>
+                        <TouchableOpacity onPress={() => navigation.navigate('Show', {id: item.id})}>
```

Comment:
used touchable opacity wrapped around the <view> and onPressed to show the blog id